app/commands.js

```diff
@@ -10,16 +10,16 @@ const commands = {
   },
   'tab:new': focusedWindow => {
     if (focusedWindow) {
-      focusedWindow.rpc.emit('termgroup add req');
+      focusedWindow.rpc.emit('termgroup add req', {});
```

Comment:
The empty object is needed to avoid this crash:

![image](https://user-images.githubusercontent.com/306879/40205045-77366ed4-59df-11e8-8cac-17fcfc32d823.png)